doc/requirements.txt

```diff
@@ -0,0 +1,2 @@
+openfisca_core >= 4.2.0, < 5.0
```

Comment:
Add this piece of information as a comment in the file itself, otherwise someone else (including yourself in 6 months) will try to change it again.